scripts/accuracy_version_plot.py

```diff
@@ -30,9 +30,9 @@ def update_plot(version):
     # Save new values
     df.to_csv('input/accuracy/difference.csv', index=False)
 
-    version = sorted(list(set(df['version'])), reverse=True)
-    version = version[:6]
-    for v in version:
+    df['version'] = df['version'].apply(lambda s: s if s.startswith('v') else f'v{s}')
+    version = sorted(list(set(df['version'])), key=lambda s: list(map(int, s[1:].split('.'))), reverse=True)
```

Comment:
I would add a comment that explains whats going on:
- Convert versions to lists "v1.0.3" -> [1,0,3]
- Python sorts a list of lists via elementwise comparisons, i.e. [2,1] > [1,2], and [1,3,1] > [1,2,3]